libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/impl/EmailNotificationService.java

```diff
@@ -337,7 +338,7 @@ private MimeMessage getMimeMessage(String subject, String body, Collection<Strin
         return message;
     }
 
-    private boolean sendEmailMIME(String subject, String body, boolean status, Collection<String> recipients) {
+    private boolean sendEmailMIME(String subject, String body, Collection<String> recipients) {
```

Comment:
As we talked about on a different pull request, the status will not be part of the interface